src/Tribe/Generator/Event.php

```diff
@@ -425,6 +425,7 @@ public function add_rsvp( $event ) {
 		];
 
 		tribe( 'tickets.rsvp' )->ticket_add( $event->ID, $data );
+        add_post_meta( $event->ID, '_EventCost', '0' );
```

Comment:
```suggestion
        update_post_meta( $event->ID, '_EventCost', '0' );
```
I'd use update - as a just-in-case.